domain/rules/basic-rules/neant-rule.ts

```diff
@@ -11,12 +10,12 @@ export class NeantRule extends DiceRule {
   applyDiceRule(context: PlayTurnGameContext): RuleEffects {
     return [
       {
```

Comment:
To discuss, those 2 events say the same thing in the history.

If we add to the history the `add grelottine` event, then it becomes obvious that each time a player rolls a neant, it will have a grelottine.

Maybe we can merge those 2 events into one
"Néant - Le joueur obtient une grelottine"